src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py

```diff
@@ -40,7 +40,25 @@
 
 name = "ElasticJobParametersDB"
 
-mapping = {"properties": {"JobID": {"type": "long"}, "Name": {"type": "keyword"}, "Value": {"type": "text"}}}
+mapping = {
+    "properties": {
+        "JobID": {"type": "long"},
+        "timestamp": {"type": "date"},
+        "CPUNormalizationFactor": {"type": "long"},
+        "MatcherServiceTime": {"type": "long"},
```

Comment:
This could be removed, it is not really useful.

Reply:
The mapping you mean?

Reply:
No I would really remove reporting this parameter.